pom.xml

```diff
@@ -868,6 +868,12 @@
         <version>1.2.3</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>io.github.artsok</groupId>
+        <artifactId>rerunner-jupiter</artifactId>
+        <version>2.1.6</version>
+        <scope>test</scope>
+      </dependency>
```

Comment:
We need to include the LICENSE file of this, isn't it?

Reply:
Doesn't look like pure test dependencies are included there.